### DISCUSSION TITLE
Is Authoring Mode Supported for Component Definitions?

### DISCUSSION THREAD

**Discussion by aj-stein-nist** [oscal-compass/compliance-trestle#1028] [Q&A] [answered]
It is called out [in the tutorials section](https://ibm.github.io/compliance-trestle/demonstrations-content/#converting-a-spreadsheet-into-a-component-definition), but is it supported? If not, is it on the roadmap explicitly or not linked to issues?

**Comment by aj-stein-nist** [comment]
@fsuits, am I missing something in the documentation, or is this not currently supported?

**Comment by fsuits** [comment, marked as answer]
Hi @aj-stein-nist, it is supported as in the demo in terms of converting from a spreadsheet, but we don't have tools based on markdown, if that is what you're looking for.  For the current release we are focusing on ssp authoring with the existing tools and their extensions.  We can look into markdown support for component definitions if you think that would be a popular need.

**Reply by aj-stein-nist** [reply]
Ok, thanks for the response, @fsuits!
